package.json

```diff
@@ -27,29 +27,39 @@
     "Google Cloud Vision API"
   ],
   "scripts": {
+    "test": "c8 mocha build/test",
+    "samples-test": "cd samples/ && npm link ../ && npm install && npm test && cd ../",
+    "lint": "gts fix && eslint --fix samples/*.js",
     "docs": "jsdoc -c .jsdoc.js",
-    "lint": "eslint '**/*.js'",
-    "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
-    "system-test": "mocha system-test/*.js smoke-test/*.js --timeout 600000",
+    "system-test": "mocha build/system-test",
+    "fix": "gts fix",
     "test-no-cover": "mocha test/*.js",
-    "test": "nyc mocha",
-    "fix": "eslint --fix '**/*.js'",
     "docs-test": "linkinator docs",
     "predocs-test": "npm run docs",
-    "prelint": "cd samples; npm link ../; npm i"
```

Comment:
Thinking we need to get this line added to the generator :)

Reply:
yes, this line is needed for lint test, and we should add it to the template. Thanks for the suggestion!

Reply:
But I'm thinking some libraries does not `samples/` or samples-test, it might break the `lint` scrip. Any advice whether we want to put it in the template? @alexander-fenster

Reply:
This is harder than it seems. We don't want to add _just_ this line, because not all libraries have samples. If we auto-generate a dummy samples package (just one `samples/package.json`), we'll need to make sure to exclude it in all real `synth.py` scripts, otherwise, the actual samples will be overwritten. Let's talk about it.

I'm leaning to having a default list of files (set in `synthtool`) that will be always excluded (rather than have this list in each `synth.py`), such as `README.md`, `package.json`, etc. If we have this list first, then we can make this change safely.  What do you think @SurferJeffAtGoogle @JustinBeckwith ?

Reply:
> This is harder than it seems. We don't want to add just this line, because not all libraries have samples. 

I think it's reasonable to say that every library we ship should at least have a quickstart.  They generally don't take much time, and go a long way to a.) making the library look real and loved and b.) provide just a little end to end test coverage.  

That having been said - we have a piece of code in synthtool that copies templates, and it's specific to nodejs I think.  I have no problem hard coding things in there for files we know we never wanna touch.

Reply:
process-wise, we always enable sample tests on a new library, so having a `quickstart.js` is pretty much a requirement. We still write these in JavaScript, so it's nice to have the `eslint`.

we'll soon be able to migrate towards `gts` for this.